src/collections/_documentation/platforms/python/gcp_functions.md

```diff
@@ -0,0 +1,54 @@
+---
+title: GCP Functions
```

Comment:
Please use WIZARD/ENDWIZARD tags like in other integrations and add it to `platforms.yml` and `platform_icons.yml`:

* https://github.com/getsentry/sentry-docs/blob/93777fd7c3a3d4d8ce4b84c21a32953bb58f51fa/src/_data/platform_icons.yml#L91-L94
* https://github.com/getsentry/sentry-docs/blob/93777fd7c3a3d4d8ce4b84c21a32953bb58f51fa/src/_data/platforms.yml#L220-L227

All of those measures will allow the user to search for serverless in the setup wizard within the Sentry UI and get the tagged excerpt from this page.